### ISSUE TITLE
Error occurred when executing INPAINT_ApplyFooocusInpaint:  Sizes of tensors must match except in dimension 1. Expected size 1 but got size 2 for tensor number 1 in the list.

### ISSUE THREAD

**Issue by meetedlike** [Acly/comfyui-inpaint-nodes#34]
Error occurred when executing INPAINT_ApplyFooocusInpaint:

Sizes of tensors must match except in dimension 1. Expected size 1 but got size 2 for tensor number 1 in the list.

File "D:\ComfyUI_windows_portable\ComfyUI\execution.py", line 151, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\ComfyUI_windows_portable\ComfyUI\execution.py", line 81, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\ComfyUI_windows_portable\ComfyUI\execution.py", line 74, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
File "D:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui-inpaint-nodes\nodes.py", line 155, in patch
feed = torch.cat([latent_mask, latent_pixels], dim=1)
<img width="956" alt="inpaint" src="https://github.com/Acly/comfyui-inpaint-nodes/assets/157203596/5a661f32-1855-4015-994b-ea542d8f605a">

**Comment by meetedlike** [comment]
Hello author, you have not replied to or addressed the question I raised. Is it because it is very difficult to handle?

**Comment by Acly** [comment]
It looks related to batch frame (video) generation. There are few other issues about it. It probably requires special support, I haven't had time or incentive to look into videos so far.

**Comment by meetedlike** [comment]
You are right, because I want to expand the image of the video, so I think the effect you have is the best when I use it on a single image. So I was wondering if it can be applied to the video. However, I found that the current processing logic is not suitable when it is used on the video because it cannot handle batch images. I hope you can do this when you have time. I think the effect will also be very good, especially when combined with animatediff, the effect is even better. Thank you for your plugin
